src/components/data/index.js

```diff
@@ -155,6 +55,10 @@ const data = {
     {
       text: categories.mentor,
       backgroundColor: '412986'
+    },
+    {
+      text: categories.code,
+      backgroundColor: 'B26F20'
```

Comment:
Chose a random color from a category not in use, but happy to change this to whatever.